jobwatch/skawatch.py

```diff
@@ -53,6 +59,26 @@ def __init__(self, task, maxage=1, errors=jobwatch.ERRORS, requires=(),
                                           maxage=maxage)
 
 
+class KadiWatch(JobWatch):
+    def __init__(self, task, filename, maxage=1):
+        self.type = 'Application Data'
+        super(KadiWatch, self).__init__(task, filename, maxage=maxage)
+
+    @property
+    def filelines(self):
+        return []
+
+    @property
+    def age(self):
+        if not hasattr(self, '_age'):
+            dwells = events.dwells.filter(-20)
+            last_dwell = dwells[len(dwells) - 1]
+            self.filetime = CxoTime(last_dwell.start).unix
+            self.filedate = time.ctime(self.filetime)
```

Comment:
This is clunky but I didn't see a driver to update the formatting of the rest of the time strings.